photon-core/src/main/java/org/photonvision/common/hardware/StatusLED.java

```diff
@@ -45,4 +45,14 @@ public StatusLED(List<Integer> statusLedPins) {
             blueLED = new CustomGPIO(statusLedPins.get(2));
         }
     }
+
+    public void setRGB(boolean r, boolean g, boolean b) {
+        // System.out.printf("DEBUG setting LED's to %b %b %b\n", r, g, b);
```

Comment:
guh

Reply:
gotta debug. I"m gonna snag hardware from our warehouse today to confirm it's actually pumping out values.